profiles/coreteam.json

```
@@ -0,0 +1,117 @@
+{
+ "slug": "69cf48da-9394-4b5f-b484-460b8d08464b",
+ "coreteam": [
+    {
+        "uid": "377b3207-553d-4546-a898-b808431e63dc",
```

```suggestion
        "uuid": "377b3207-553d-4546-a898-b808431e63dc",
```
please change the `UID` to `uuid`